tests/store/test_rest_store.py

```diff
@@ -54,7 +54,7 @@ def test_failed_http_request(self, request):
     @mock.patch('requests.request')
     def test_response_with_unknown_fields(self, request):
         experiment_json = {
-            "experiment_id": 1,
+            "experiment_id": "1",
```

Comment:
parametrizing this file to test rest_store with both int and string experiment_ids

Reply:
the edit wil have to be after the message_to_json to avoid protobuf deserialization errors

Reply:
We will be testing against the previous rest store manually to avoid carrying over unused code into the newer versions